src/modelling_utilities/get_dynamic_images_from_parametric_images.cxx

```diff
@@ -22,16 +23,26 @@
   \ingroup utilities
   \brief Multiplies Parametric Images with the Model Matrix creating Dynamic Images
   \author Charalampos Tsoumpas  
+  \author Richard Brown
 
   \par Usage:
   \code 
-  get_dynamic_images_from_parametric_images output_parametric_image input_dynamic_image [par_file] 
+  get_dynamic_images_from_parametric_images output_parametric_image input_dynamic_image [par_file] [output_format_par_file]
```

Comment:
can't be correct. where's the optional template? 2 options would need to be as `[option1 [ option2]]`. Needs to be backwards compatible as well. I haven't checked how it was...